src/dodal/beamlines/i04.py

```diff
@@ -303,6 +303,7 @@ def undulator(
         wait_for_connection,
         fake_with_ophyd_sim,
         bl_prefix=False,
+        id_gap_lookup_table_path="/dls_sw/i03/software/daq_configuration/lookup/BeamLine_Undulator_toGap.txt",
```

Comment:
@DominicOram I added the path from i03 as I don't see the undulator_dcm at i04.

if dcm is not present at i04 this parameter kind of does not make sense, we could make the lookup table path optional and then not allow for the method to update it proceed if the object does not have that path property

Reply:
The correct location is `/dls_sw/i04/software/gda/config/lookupTables/BeamLine_Undulator_toGap.txt`

Reply:
applied